Editor/IBMCloudSignup.cs

```diff
@@ -1,5 +1,5 @@
 ﻿/**
-* (C) Copyright IBM Corp. 2018, 2019.
+* (C) Copyright IBM Corp. 2018, 2020.
```

Comment:
Noting that this year is updated to 2020 since the copyrights were normalized in the [last release](https://github.com/watson-developer-cloud/unity-sdk/pull/614/files)